Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -115,8 +115,8 @@ class BrowserViewController: UIViewController {
     var pendingRequests = [String : URLRequest]()
 
     // This is set when the user taps "Download Link" from the context menu. We then force a
-    // download of the next request through the `WKNavigationDelegate` that matches this URL.
-    var pendingDownloadURL: URL? = nil
+    // download of the next request through the `WKNavigationDelegate` that matches this web view.
+    var pendingDownloadWebView: WKWebView? = nil
```

Comment:
this is storing a strong reference to a webview. Looking at how we clear this webview I dont think it'll be an issue but just wanted to double check.